homeassistant/components/synology/camera.py

```diff
@@ -42,6 +42,12 @@
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up a Synology IP Camera."""
+    _LOGGER.warning(
+        "The Synology integration is deprecated."
+        " Please use the Synology DSM integration"
```

Comment:
We should add the release in which we will remove this integration.

Reply:
I put 0.120.0. Not sure if it is a good choice.

Reply:
I think you can go for 0.118.0

It's mostly v+2 @springstan , right ?

Reply:
Yeah I have also mostly seen v+2.

Reply:
Changed to 0.118.0